### DISCUSSION TITLE
iOS backspace handler on an empty editor

### DISCUSSION THREAD

**Discussion by MagTuxGit** [singerdmx/flutter-quill#514] [Mobile]
I faced the issue when `QuillController.handleDelete()` isn't called on iOS when backspace pressed on an empty editor.
I found explanation [here](https://github.com/flutter/flutter/issues/14809)
Also I found a solution [here](https://medium.com/super-declarative/why-you-cant-detect-a-delete-action-in-an-empty-flutter-text-field-3cf53e47b631)
This solution works, I've implemented it in my project.
I have an idea to incapsulate this into the package so `QuillController.handleDelete()` will work.
How do you think?

**Comment by qwer122120043** [comment]
Why onDelete doesn't trigger when I press backspace on my android phone?

Looking forward to your reply, thank you very much!
